backend/routes/dRoutes.jsx

```diff
@@ -0,0 +1,42 @@
+const { PrismaClient } = require('@prisma/client')
+const prisma = new PrismaClient()
+
+app.get('/books', async (req, res) => {
```

Comment:
noob q, why do we use "books" as the directory name? It's supposed to store "routes" if I understand correctly

Reply:
Done